### ISSUE TITLE
Excluded Group Ids and Exclude Words or Phrases not working

### ISSUE THREAD

**Issue by jabby60** [JohnnyRS/PandaCrazy-Max#42]
First, great extension and thank you for it. Can't get search option in neither PandaUI nor SearchUI to abide by excluded group ids, words or phrases in latest version 0.9.30.0

**Comment by mm2048** [comment]
Just to clarify, is 0.9.30 the first version you have tried or was it working in 0.9.29 and after upgrading to 0.9.30 they stopped working for you?

**Comment by jabby60** [comment]
Recently just tried using keywords/phrases for the first time on the previous version and it didn't work. Then updated to the newest version and it still doesn't work. Also noticed that the SearchUI won't work if you have another search going on the PandaUI. Not sure if that's intended, but likely doesn't matter if all searches can be done in the SearchUI.

**Comment by JohnnyRS** [comment]
I did some tests right now with exclude words and include words and it seems to work with the current version with all triggers. I did find that the groupID exclude may not be working. If you have a duplicate search on the pandaui that the searchui has it may not work but otherwise both should be working. Also remember when adding include or exclude words it will only work on newer HITs not older HITs. I'll keep testing to figure out what may be going on.

**Comment by jabby60** [comment]
You're right. I forgot it won't work on older Hits. I was trying to exclude some older HIts from a requester and it kept accepting them. Thanks
